alacritty_terminal/src/config/mod.rs

```diff
@@ -108,11 +108,10 @@ pub struct Config<T> {
     #[serde(default, deserialize_with = "failure_default")]
     pub cursor: Cursor,
 
-    /// Enable experimental conpty backend instead of using winpty.
-    /// Will only take effect on Windows 10 Oct 2018 and later.
+    /// Disable conpty backend and use winpty even if conpty is available.
```

Comment:
You've doubled down on being super precise here, which I don't think is necessary. If ConPTY is disabled, it's implied that WinPTY isn't and the other way around.

The following shortens this a bit while conveying the same information:
```suggestion
    /// Use WinPTY backend even if ConPTY is available.
```